eessi_bot_job_manager.py

```diff
@@ -54,28 +53,18 @@ def get_current_jobs(self):
         username = os.getlogin()
 
         squeue_cmd = "%s --long --user=%s" % (self.poll_command, username)
-        log(
-            "get_current_jobs(): run squeue command: %s" % squeue_cmd,
-            self.logfile,
```

Comment:
@Hafsa-Naeem As @trz42 pointed out, we should enhance `run_cmd` so a `logfile` can be passed into it, which is used in the `log` statements there:

```python
run_cmd(squeue_cmd, ..., logfile=self.logfile)

Reply:
Also add a test if you enhance `run_cmd`:

```python
log_file = os.path.join(tmpdir, "log.txt")
run_cmd("echo hello", "test in file", tmpdir, log_file=log_file)
with open(log_file, "r") as fp:
    assert "test in file" in fp.read()